src/app/modules/item/pages/item-edit-content/item-edit-content.component.ts

```diff
@@ -9,6 +10,10 @@ import { FormGroup } from '@angular/forms';
 export class ItemEditContentComponent {
   @Input() parentForm?: FormGroup;
 
-  constructor() {}
+  itemData$ = this.itemDataSource.itemData$;
```

Comment:
If we do not use routing, we should not use the datasource. It should be passed by input.

Reply:
Done

Reply:
Fixed